geozero/src/gdal/gdal_writer.rs

```diff
@@ -3,6 +3,8 @@ use crate::{CoordDimensions, FeatureProcessor, GeomProcessor, PropertyProcessor}
 use gdal::vector::Geometry;
 use gdal_sys::OGRwkbGeometryType;
 
+use super::GdalError;
```

Comment:
I would change this to this to highlight (and test) that the error enum is accessible from the root - consistent with how lib users will use it.  So either `use crate::GdalError;` or `use crate::gdal::GdalError;`